pkg/chart/dependency.go

```diff
@@ -77,9 +85,18 @@ func syncDependencies(chartPath string, sourceRepo *api.Repo, target *api.Target
 	}
 
 	if !missingDependencies {
-		klog.V(3).Info("Updating requirements.yaml file...")
-		if err := updateRequirementsFile(chartPath, lock, sourceRepo, target); err != nil {
-			return errors.Trace(err)
+		klog.V(3).Info("Updating Chart.yaml file...")
+		switch apiVersion {
+		case "v1":
```

Comment:
I would add a `types.go` file stating this kind of constants:

```golang
package chart

const (
  API_V1 string = "v1"
  API_V2 string = "v2"
)
```

you can use this file to define common constants such as "Chart.yaml"